src/Directory.Packages.props

```diff
@@ -3,6 +3,9 @@
         <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     </PropertyGroup>
     <ItemGroup>
+        <PackageVersion Include="Aspire.Hosting.AppHost" Version="9.4.0" />
```

Comment:
Will add these packages to our internal feed, after which you wont need nuget.org in the Nuget.config